src/openrct2/windows/guest_list.c

```diff
@@ -706,14 +706,14 @@ static void window_guest_list_scrollpaint(rct_window *w, rct_drawpixelinfo *dpi,
 
         // For each guest
         FOR_ALL_GUESTS(spriteIndex, peep) {
-            peep->flags &= ~(SPRITE_FLAGS_PEEP_FLASHING);
```

Comment:
I imagine this will remove all uses of `SPRITE_FLAGS_PEEP_FLASHING`, can you please leave a comment next to where it is declared, explaining why is it not used anymore?